packages/stream/get-streams/get-streams.js

```diff
@@ -0,0 +1,57 @@
+import { InitializationInstance } from '../../app/init/init.js'
+import { Internal } from '../../internal/index.js'
+
+/**
+ * @typedef FetchResponse
+ * @property {object} status -- A status response
+ * @property {Array<any>} data -- A return data as per endpoint
+ */
+
+/**
+ * A get list of streams module based on the API Key (per project)
+ *
+ * @function
+ * @param {InitializationInstance} initObject -- initialization object
+ * @returns {Promise<FetchResponse>} returns the restructured data which content status & list of streams data
+ * @throws {Error}
+ */
+export const getStreams = async (initObject) => {
+  if (!(initObject instanceof InitializationInstance)) {
+    throw new TypeError(
+      'Failed to process because initialization is not valid. Please provide required initialization argument which is the initialization instance returned by the init() function'
+    )
+  } else {
+    let fetchResponse = await Internal.fetchHttp({
+      url: `${Internal.config.api.base_url}/${Internal.config.api.version}/streams/`,
+      token: initObject.config.api_key,
+      method: 'GET',
+    }).catch((error) => {
+      return error
```

Comment:
Same with [this PR](https://github.com/inlivedev/inlive-js-sdk/pull/12/files#r973819622). You can handle the error here or `throw` the error and catch it on another place. But please don't return the error.

Reply:
@faiq-naufal Pardon me I skip this first, based on same as [this PR](https://github.com/inlivedev/inlive-js-sdk/pull/12#discussion_r974039900) consideration.